apps/wfo-ui-surf/components/WfoAvatar/WfoAvatar.tsx

```diff
@@ -27,20 +35,24 @@ export const WfoAvatar = ({ stepStatus, icon }: WfoAvatarProps) => {
                 return [updateIconStyle];
             case ServiceTicketLogType.CLOSE:
                 return [closedIconStyle];
```

Comment:
Is the default case the same as this one now?

Reply:
The CLOSE is gray and the default case is everywhere usually gray as well. Wanted to make it more explicit, instead of skipping the `case ServiceTicketLogType.CLOSE:` and just use straight default